Dockerfile

```diff
@@ -1,23 +1,25 @@
-FROM openshift/origin-release:golang-1.10 as builder
-COPY . /go/src/github.com/openshift/openshift-tuned/
-RUN cd /go/src/github.com/openshift/openshift-tuned && make build
-
-FROM centos:7
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/openshift-tuned
+COPY . .
+RUN make build
 
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 ENV APP_ROOT=/var/lib/tuned
-ENV PATH=${APP_ROOT}/bin:${PATH} HOME=${APP_ROOT}
+ENV PATH=${APP_ROOT}/bin:${PATH} 
+ENV HOME=${APP_ROOT}
 WORKDIR ${APP_ROOT}
+COPY --from=builder /go/src/github.com/openshift/openshift-tuned/openshift-tuned /usr/bin/
```

Comment:
openshift-tuned binary doesn't exist, therefore this step will fail.  Why did you add this @smarterclayton ?  I could rename tuned-wait to openshift-tuned, and remove tuned-wait completely if I'm reading your intentions here right.  Thanks.